EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php

```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```

Comment:
Wouldn't a separate PR lose its context of the removal, though? That would make the removal seem malicious or without point. Right?